Sources/TuistCore/Models/Plugins.swift

```diff
@@ -0,0 +1,20 @@
+import Foundation
+
+/// A model which contains all loaded plugin representations.
+public struct Plugins: Equatable {
```

Comment:
How is this model being used?

Reply:
Currently, we aren't using it as this PR only sets up the models. But the idea is that we would convert each `Plugin` manifest and when we have them loaded we then transform them into `Plugins` which is an easier to use/pass around model. It will contain all the plugin types, right now it only has the project description helpers.